crates/bitwarden/src/client/encryption_settings.rs

```diff
@@ -28,30 +28,39 @@ impl std::fmt::Debug for EncryptionSettings {
 impl EncryptionSettings {
     #[cfg(feature = "internal")]
     pub(crate) fn new(
-        auth: &AuthSettings,
+        login_method: &LoginMethod,
```

Comment:
This should probably take a `UserLoginMethod` directly, it doesn't make sense for `ServiceAccount`, which would use `new_single_key` instead